packages/modelviewer.dev/styles/examples.css

```diff
@@ -311,6 +311,11 @@ h1 span {
   margin: var(--snippet-offset);
 }
 
+.content .quick-start-example example-snippet {
```

Comment:
On android, the h4 text seems to be about the same size as the code snippet text (good). On my mac, it's quite a bit larger, nearly as big as the h3 size. Is this a `px` vs `em` thing?

Reply:
**"it's"** quite a bit larger -- what exactly is the it's referring to? the code snippet or h4?

if you are referring to h4, this is due to the sizes being scaled as the screen sizes change (you'll see this if you scale the screen width down)

this is currently what happens on dev as well and is due to: @media only screen and (max-width: 800px), inside it, the text is changing for any window smaller than 800 px (like an android)

Reply:
Yeah, the h4 is bigger on desktop, relative to both the h3 and the code snippet. Seems like they should all be scaled together, maintaining relative size, more or less for mobile vs desktop.

Reply:
I don't think that's exactly right. So em is apparently: Relative to the font-size of the element (2em means 2 times the size of the current font).

So, the difference between smaller numbers is more obvious than higher numbers, i.e. 98 and 100 are close but 1 and 3 would be drastic, relatively. 

The desktop has----------------1.4em for h3 and 1.2em for h4
The sub-800px width-----------1.2em for h3 and 1.0em for h4

So, yes the desktops would look closer in size, but the h4 isn't bigger.

Reply:
Okay, no biggy.

Reply:
kk